estate/models/estate_property_offer.py

```diff
@@ -48,6 +49,14 @@ def _compute_deadline(self):
             else:
                 rec.deadline = False
 
+    @api.depends('property_id', 'res_partner_id')
```

Comment:
Il faut être le plus précis possible dans les `depends`
```suggestion
    @api.depends('property_id.name', 'res_partner_id.name')
```